src/linker/Linker.Steps/SweepStep.cs

```diff
@@ -63,6 +63,17 @@ protected override void Process ()
 			foreach (var assembly in assemblies)
 				UpdateAssemblyReferencesToRemovedAssemblies (assembly);
 
+			// Update scopes before removing any type forwarder.
+			foreach (var assembly in assemblies) {
+				switch (Annotations.GetAction (assembly)) {
+				case AssemblyAction.CopyUsed:
```

Comment:
Updating scopes in a copyused assembly is new, but I think this is a good change.

Reply:
On second thought it might be best to leave it as before purely for back-compat. Some scenarios might depend on the old behavior that copies "copyused" assemblies in some cases (when there were no removed references).

Reply:
Thanks. Makes sense -- I changed this so we keep same behavior for copyused assemblies.